gnome-shell/src/gnome-shell-sass/_drawing.scss

```diff
@@ -148,7 +148,7 @@
     color: $tc;
     text-shadow: 0 1px black;
     icon-shadow: 0 1px black;
-    box-shadow: inset 0px 0px 0px 1px $selected_bg_color;
+    box-shadow: inset 0px 0px 0px 2px $selected_bg_color;
```

Comment:
We decided to use a 1px focus border for button focus.
But I found an issue in the Gtk theme with 1px focus rings, so if @madsrh @clobrano and @ubuntujaggers  agree, we can make all focus borders again 2px like in the upstream theme

Reply:
It's ok for me

Reply:
Okay for mads and Stuart also so we keep this change and adapt the gtk theme to it